### ISSUE TITLE
Registration automation

### ISSUE THREAD

**Issue by sprx97** [Spartan97/OldTimeHockey#14]
All that work the mods do at the start of the season? Emails, league invites, account verification? There's an app for that (well not yet, I have to make it).

I should be able to create a portal to allow registrations, store info in the database, and use a whitelist to send invites for appropriate divisions to appropriate managers and a blacklist to block last year's quitters. 

Super important because it could also be used for RCJ and SOS.

**Comment by sprx97** [comment]
Also should be able to handle waitlisting and waitlist invites.

Scrape FF for inactives list (already done)
1. Kick them (this is the part I'm unsure of - would have to get the bot to login as OTHAdmin)
2. Cross-reference registration
3. Send invite and update waitlist.

**Comment by sprx97** [comment]
Kinda in progress 😲

**Comment by sprx97** [comment]
Currently blocked on user accounts
